examples/csrf/crosssite/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2'
+version: '3.7'
```

Comment:
@baremaximum It would be great in a follow up if you could standardize the compose version:

- examples/csrf/crosssite/docker-compose.yml
- examples/grpc-bridge/docker-compose-protos.yaml
- examples/csrf/samesite/docker-compose.yml